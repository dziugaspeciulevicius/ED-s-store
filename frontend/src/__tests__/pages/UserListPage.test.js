import React from "react";
import { shallow } from "enzyme";
import "../../setupTests";
import UserListPage from "../../pages/admin-panel/tab-panels/UserListPage.component";

jest.mock("../../pages/UserListPage.component");

describe("<UserListPage />", () => {
  it("Renders <UserListPage /> component correctly", () => {
    // shallow creates instance of a component
    expect(shallow(<UserListPage />)).toMatchSnapshot();
  });
});
