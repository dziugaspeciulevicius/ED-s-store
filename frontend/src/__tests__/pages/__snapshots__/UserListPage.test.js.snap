// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserListPage /> Renders <UserListPage /> component correctly 1`] = `ShallowWrapper {}`;
